setup.py

```diff
@@ -32,21 +35,24 @@
 with open("README.md") as readme_file:
     long_description = readme_file.read()
 
-strict_requirements = {
-    "numba": "numba>=0.4.8",
-    "matplotlib": "matplotlib==3.2.2",
-}
-with open("requirements.txt") as requirements_file:
-    requirements = []
-    # for line in requirements_file:
-    #     # TODO, this should be a proper regex parsing
-    #     requirement, version = line.split("==")
-    #     if requirement not in strict_requirements:
-    #         requirements.append(requirement)
-    #     else:
-    #         requirements.append(strict_requirements[requirement])
-    for line in requirements_file:
-        requirements.append(line)
+
```

Comment:
While very functional, the `AlphaTemplate` version is probably somewhat cleaner, more readable and easier to update. Now would probably be the right time to adhere to that standard instead of AlphaTims.